contracts/V1/StateChannelDiamondProxy/DisputeManagerFacet.sol

```diff
@@ -54,20 +53,19 @@ contract DisputeManagerFacet is StateChannelCommon {
         DisputeAuditingData memory disputeAudit
     ) public returns (bool isSuccess, address[] memory slashParticipants) {
 
-        address challenger = msg.sender;
         // check if the commitment of dispute is available
         bytes32 memory disputeCommitment = keccak256(abi.encodePacked(
             dispute,
             block.timestamp
```

Comment:
The block.timestamp is used when making the commitment on-chain. Here you have to provide that timestamp, since it needs to be the same timestamp for the hash (commitment) to match. Feel free to extend AuditingData with it.